README.md

```diff
@@ -1,23 +1,12 @@
 # embedded-common
-This repository contains the Sensirion i2c abstraction layer and commonly shared
-code. It is a dependency of the Sensirion embedded drivers and it should not be
-necessary to clone it independently, as we recommend to download the repository
-recursively from the respective projects.
+This repository contains the Sensirion i2c hardware abstraction layer and
```

Comment:
s/i2c/I2C/

Reply:
I wanted to change it, but it's consistent all over..